src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_CSharp_Nested_HTML.semantic.txt

```diff
@@ -7,11 +7,11 @@
 0 1 3 15 0 //keyword
 0 4 1 8 0 //variable
 0 2 1 21 0 //operator
-0 1 1 43 0 //punctuation
+0 2 1 43 0 //punctuation
```

Comment:
These values also needed to be updated, I manually verified that they are correct